app/views/static/feedback.scala.html

```diff
@@ -0,0 +1,21 @@
+@*****************************************************************************
+ * Page to display a form to gather user comments                            *
+ * which are emailed to the site administrator                               *
+ * Copyright (c) 2015 MIT Libraries                                          *
+ *****************************************************************************@
+ @(fbform: Form[(String,Boolean,String)])@*(implicit hubContext: HubContext, flash: Flash)*@
+ @import helper._
```

Comment:
Adding `@import helper.twitterBootstrap._` in addition to the existing `@import helper._` will generate some marginally better HTML.
https://www.playframework.com/documentation/2.0/ScalaFormHelpers#Twitter-bootstrap-field-constructor

It'll probably eventually be worth creating our own form helper to take advantage of some of the more complex bootstrap form options. I'm partial to the form-horizontal form options aesthetically:
http://getbootstrap.com/css/#forms-horizontal